requirements/edx-sandbox/py38.txt

```diff
@@ -4,6 +4,10 @@
 #
 #    make upgrade
 #
+--index-url http://edx.devstack.devpi:3141/root/pypi/+simple/
+--extra-index-url https://pypi.python.org/simple
+--trusted-host edx.devstack.devpi
```

Comment:
@irfanuddinahmad, you still have these in `scripts/xblock/requirements.txt`. I suppose that ` --extra-index-url` is also redundant. FYI, you don't need to run requirement updates locally - you can add an extra dependency to your branch and then use the "Run workflow" option from the https://github.com/openedx/edx-platform/actions/workflows/upgrade-python-requirements.yml page to update them.

Also, it would be nice to mention adding the `federated-content-connector` in the commit message.